<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGraphDialog</class>
 <widget class="QDialog" name="AddGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add graph</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/chart-add</normaloff>:/icons/chart-add</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="gbOther">
     <property name="title">
      <string>Other</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="cbBarometricPressure">
        <property name="text">
         <string>Barometric Pressure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbRainfall">
        <property name="text">
         <string>Rainfall</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="gbHumidity">
     <property name="title">
      <string>Humidity</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbHumidity">
        <property name="text">
         <string>Humidity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbInsideHumidity">
        <property name="text">
         <string>Inside Humidity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QGroupBox" name="gbTemperature">
     <property name="title">
      <string>Temperature</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbTemperature">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbInsideTemperature">
        <property name="text">
         <string>Inside Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbApparentTemperature">
        <property name="text">
         <string>Apparent Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbWindChill">
        <property name="text">
         <string>Wind Chill</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDewPoint">
        <property name="text">
         <string>Dew Point</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QGroupBox" name="gbWind">
     <property name="title">
      <string>Wind</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cbAverageWindSpeed">
        <property name="text">
         <string>Average Wind Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbGustWindSpeed">
        <property name="text">
         <string>Gust Wind Speed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbWindDirection">
        <property name="text">
         <string>Wind Direction</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the additional graphs you would like to add to the chart. To remove a graph select it in the chart, right click and choose the remove graph option.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbTemperature</tabstop>
  <tabstop>cbInsideTemperature</tabstop>
  <tabstop>cbApparentTemperature</tabstop>
  <tabstop>cbWindChill</tabstop>
  <tabstop>cbDewPoint</tabstop>
  <tabstop>cbHumidity</tabstop>
  <tabstop>cbInsideHumidity</tabstop>
  <tabstop>cbBarometricPressure</tabstop>
  <tabstop>cbRainfall</tabstop>
  <tabstop>cbAverageWindSpeed</tabstop>
  <tabstop>cbGustWindSpeed</tabstop>
  <tabstop>cbWindDirection</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddGraphDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddGraphDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
